File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/java/chunk_D/f16p1_V3_boxedreturn.java
import java.util.ArrayList;
import java.util.List;

class Main {
    static Object sumSequence() {
        // BEGIN PROBE F16.P1
        var xs = new ArrayList<>(List.of(1, 2, 3));
        int total = xs.stream().reduce(0, Integer::sum);
        return total;
        // END PROBE F16.P1
    }

    public static void main(String[] args) {
        Object r = sumSequence();
        System.out.println(r.getClass().getName() + " " + r);
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/kotlin/chunk_D/f16p1_V3_boxedreturn.kt
fun probe(): Any {
    // BEGIN PROBE F16.P1
    val xs = mutableListOf(1, 2, 3)
    val total = xs.sum()
    return total
    // END PROBE F16.P1
}

fun main() {
    val r = probe()
    println(r.javaClass.toString() + " " + r)
}
